⚙ Turn off `sort-imports` ESLint rule of config

## Overview

* Turn off `sort-imports` ESLint rule of config

## Note

* The rule makes a lint error

* Code
  
    ```js
    import core from '../../rules/core'
    import stylisticJs from '../../rules/plugins/stylistic/js'
    import stylisticPlus from '../../rules/plugins/stylistic/plus'
    import jestPlugin from '../../rules/plugins/jest'
    import jsdocPlugin from '../../rules/plugins/jsdoc'
    ```
  
* Lint log

    ```sh
    /Users/steweucen/_open-reach-tech/_dev/_oss/eslint-rules-default/index.js
      4:1  error  Imports should be sorted alphabetically  sort-imports
    
    /Users/steweucen/_open-reach-tech/_dev/_oss/eslint-rules-default/tests/__tests__/main-exports.js
      4:1  error  Imports should be sorted alphabetically  sort-imports
    ```

--- a/eslint.config.js
+++ b/eslint.config.js
@@ -1405,7 +1405,7 @@ export default [
         'first',
       ],
       'sort-imports': [
-        'error',
+        'off',
         {
           ignoreCase: false,
           ignoreDeclarationSort: false,

